src/handlebars/index.js

```diff
@@ -13,7 +13,8 @@ module.exports = function (conf) {
 
 	return handlebars(app, {
 		partialsDir: [
-			directory + (options.viewsDirectory || '/views') + '/partials'
+			directory + (options.viewsDirectory || '/views') + '/partials',
+			directory + ('/node_modules/@financial-times')
```

Comment:
If we made it more specific we'd have to do a glob to find the specific ones and change the referencing to be less specific.

With the specific reference in the template to the partial I don't think that this should be costly at start up as should only be invoked when looking for a specific partial.